documentation/docs/deploy/flags.md

```diff
@@ -28,6 +28,7 @@ Parameters/Flags can be defined in 2 ways:
 |debug|Boolean|Enables or disables starting processes in debug mode. It overrides any debug setting in configuration. If not set debugging is enabled.|
 |databasePort|Any available port|The port on which database is going to be started.|
 |logSql|true/false|Enables printing of SQL queries executed on the server|
+|terraformProvider|`aws`<br>`azure`|Terraform cloud provider options|
```

Comment:
Is it really true? I don't see in the code that it is possible to configure Azure.